ui/src/containers/ClusterMonitoring.js

```diff
@@ -238,7 +255,16 @@ const ClusterMonitoring = props => {
           </Tooltip>
         </RightClusterStatusContainer>
       </ClusterStatusTitleContainer>
-
+      {clusterStatus.value === CLUSTER_STATUS_UNKNOWN &&
```

Comment:
Instead of checking this `UNKNOWN` status, why not use the `monitoring.isPrometheusApiUp` boolean that already exists in the state?

`UNKNOWN` could exist while a query is in progress, for instance.